### PR TITLE
OTP 18 changes

### PR DESCRIPTION
**Pull request by fenek**

### CODE REVIEW

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'exml' is not using appropriate protocol, please change this to something like 'https://.*'

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'wsecli' is not using appropriate protocol, please change this to something like 'https://.*'

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'fusco' is not using appropriate protocol, please change this to something like 'https://.*'

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'base16' is not using appropriate protocol, please change this to something like 'https://.*'

File: src/escalus_history_h.erl [hunk 1]
```diff
@@ -10,9 +10,15 @@
 	 handle_event/2,
```
Comment (gadgetci) [inline src/escalus_history_h.erl]: According to **Elvis**:
> Line 10 has a tab at column 0.

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'meck' is not using appropriate protocol, please change this to something like 'https://.*'

**Comment by gadgetci** [comment]
According to **Elvis**:
> Dependency 'mustache' is not using appropriate protocol, please change this to something like 'https://.*'

**Comment by michalwski** [comment]
How about adding OTP 18 to escalus's .travis.yml file?

**Comment by fenek** [comment]
@michalwski Done but it seems it won't succeed until PR chain will reach MIM itself eventually.

**Comment by michalwski** [comment]
Right, because of Mongoose... I forgot about this.

**Comment by michalwski** [comment]
So let's revert the travis change and wait until Mongoose is updated.

**Comment by fenek** [comment]
Reverted.

**Pull request merged**